### DISCUSSION TITLE
How to Update Sumatra?

### DISCUSSION THREAD

**Discussion by AnxiousPleaseHelp** [sumatrapdfreader/sumatrapdf#4568] [General]
Hello,

I'm running Sumatra 3.4.6 64bit on my Windows 10 64bit PC.

I just downloaded the newest versions of Sumatra from here: https://www.sumatrapdfreader.org/download-free-pdf-viewer

I downloaded the 64bit installer version and the 64bit portable version.

Do I just run both of these files or do I need to uninstall my current version of Sumatra?

What happens if I keep my current version of Sumatra installed and then I also installed the newest portable version?

Also, whats the best way I can make sure that Sumatra is unable to contact the Internet? I don't want Sumatra to be able to connect to the Internet. 

I hope someone can help me! 😢😢😢

**Comment by GitHubRulesOK** [comment]
for restrictions read the help system https://www.sumatrapdfreader.org/docs/Configure-for-restricted-use

You only need the installer when using certain mixes like using the PDF previews of thumbnails or helping windows "file search" find words in the folders of PDFs .

Official Portable is fine for all other tasks as installer is only need to control system registry settings.

Having 2 versions unless YOU decide why, is not normally recommended as it just adds to confusions. You might like me keep one as default install but dozens of others for different reasons But I assure you that trips me up on occasion.
